rcl/test/rcl/test_publisher.cpp

```diff
@@ -76,6 +80,34 @@ class CLASSNAME (TestPublisherFixture, RMW_IMPLEMENTATION) : public ::testing::T
   }
 };
 
+class CLASSNAME (TestPublisherFixtureInit, RMW_IMPLEMENTATION)
+  : public CLASSNAME(TestPublisherFixture, RMW_IMPLEMENTATION)
+{
+public:
+  const rosidl_message_type_support_t * ts =
+    ROSIDL_GET_MSG_TYPE_SUPPORT(test_msgs, msg, BasicTypes);
+  const char * topic_name = "chatter";
+  rcl_ret_t ret;
```

Comment:
This wall cause `rcl_ret_t` variables called `ret` in each unit test to shadow this one. Can you just use a local instead?

Reply:
See 06caaca